finetune/scheduler.py

```diff
@@ -23,89 +60,101 @@ def __init__(self, max_models=None, config=None, reserved=750000000):
         self.max_model_size = None
         self.config = config or {}
         self.reserved = reserved
+        self.ram_max_frac = ram_max_frac
 
     def _memory_for_one_more(self):
         if self.session is None:
-            return True # First prediction run?
-            
+            return True  # First prediction run?
+
         in_use, peak = self.session.run((self.in_use_op, self.peak_mem_op))
         if self.max_above_resting is None or (peak - in_use) > self.max_above_resting:
             self.max_above_resting = peak - in_use
 
-        if self.max_model_size is None or (in_use - self.previous_in_use) > self.max_model_size:
+        if (
+            self.max_model_size is None
+            or (in_use - self.previous_in_use) > self.max_model_size
+        ):
             self.max_model_size = in_use - self.previous_in_use
 
         self.previous_in_use = in_use
+        cpu_percent = psutil.virtual_memory().percent
```

Comment:
I'm unsure whether we should be tracking memory_percent() or virtual_memory().percent here